### DISCUSSION TITLE
`prisma format` using tab indentations?

### DISCUSSION THREAD

**Discussion by catrektson** [prisma/prisma#18209] [Q&A] [answered]
Is there a way to format `schema.prisma` using tab indentations? It seems to indent using spaces by default.

Would be glad if that was possible.

**Comment by ludralph** [comment, marked as answer]
Hi @catrektson 👋 

Prisma formatting rules are fixed, just like `gofmt`.  `.prisma` files are formatted automatically and have no configuration options.
See this [section](https://www.prisma.io/docs/concepts/components/prisma-schema#auto-formatting) of the documentation

**Reply by ludralph** [reply]
Hey @catrektson,

Do you have any more questions around this issue? If no, it would be really helpful if you could quickly **mark this Discussion as _answered_** to indicate that your issue has been resolved. 

In case you do have any outstanding questions, feel free to let us know and we'd be happy to help!

**Reply by ludralph** [reply]
Hi @catrektson 

You may have missed my previous messages. Do let us know if you have any other questions on this issue.
